tests/test_transactions_reader.py

```diff
@@ -127,14 +127,17 @@ def test_transactions_qty_not_integer(testfiles_dir):
                               50.00,
                               0.0,
                               'USD')
+    # Overwrite the qty after creating the object because otherwise the object
+    # initialization will throw an error
+    transaction._qty = 'BLAH'
     transactions = transactions_to_list([transaction])
     filepath = create_csv_file(testfiles_dir,
                                "qtynotinteger.csv,",
                                transactions,
                                True)
     with pytest.raises(ClickException) as excinfo:
         TransactionsReader.get_transactions(filepath)
-    assert excinfo.value.message == "The quanitity entered 100.1 is not an integer"  # noqa: E501
+    assert excinfo.value.message == "The quantity entered BLAH is not a valid number"  # noqa: E501
 
 
 def test_transactions_price_not_float(testfiles_dir):
```

Comment:
Should this test be renamed since its not about floats anymore?

Reply:
Yes good point, I will change it.